doc/qbk/4.0.containers.qbk

```diff
@@ -0,0 +1,41 @@
+[/
+    Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/CPPAlliance/url
+]
+
+[section:containers Containers]
+
+The library provides containers for representing URLs according to
+their generic syntax for well known schemes. The three main
+URL containers are:
+
+* __url_view__: A non-owning reference to a valid URL.
+* __url__: A modifiable container for a URL.
+* __static_url__: A URL with fixed-capacity storage.
+
+These containers are meant to cover common use cases such
+as inspecting external strings with URLs, modifying
+URLs, and handling URLs without dynamic memory allocations
+when the maximum URL size is known ahead of time. The
+design goals of these containers are:
+
+* Views to inspect existing URL strings
+* No exotic character types, just low-ASCII `char`
+* Work without exceptions
+* Fast compilation
+* Avoiding allocation entirely when the maximum size is known
+* Work well on embedded devices.
```

Comment:
This sort of thing is applicable to the whole library and belongs in the Overview page not here